graph/src/runtime/asc_ptr.rs

```diff
@@ -173,6 +174,21 @@ impl<C: AscType> AscPtr<C> {
         self.0 == 0
     }
 
+    /// There's no problem in an AscPtr being 'null' (see above AscPtr::is_null function).
+    /// However if one tries to read that pointer, it should fail with a helpful error message,
+    /// this function does this error handling.
+    ///
+    /// Summary: ALWAYS call this before reading an AscPtr.
+    pub fn assert_is_not_null(&self) -> Result<(), DeterministicHostError> {
```

Comment:
Minor, but it would be better to call this `check_is_not_null`, when I see `assert` I expect that the code could panic.

Reply:
Makes sense! I'll change this rn :)

Reply:
Fixed!